doc/project/checklist.md

```diff
@@ -183,32 +183,35 @@ Review Design Change with SW: Review known "Won't Fix" bugs and "Errata".
 
 ## V1
 
-For a transition from V0 to V1, the following items are expected be completed.
-Prefix "SIM" is applicable for simulation-based DV approach only, while "FPV" is for FPV approach only.
+To transition from V0 to V1, the following items are expected be completed.
+Prefix "SIM" is applicable for simulation-based DV approach, whereas the prefix "FPV" is applicable for formal property-based verification approach.
 
 ### DV_DOC_DRAFT_COMPLETED
 
-- DV document drafted, indicating the overall DV strategy, intent and the testbench environment details with diagrams, details on TB, UVCs, checkers, scoreboard, interfaces, assertions, coverage objects (if applicable).
-- Details may be missing since most items are not expected to be fully developed at this stage.
+DV document drafted, indicating the overall DV goals, strategy, the testbench environment details with diagram(s) depicting the flow of data, UVCs, checkers, scoreboard, interfaces, assertions and the rationale for the chosen functional coverage plan.
+Details may be missing since most of these items are not expected to be fully understood at this stage.
 
-### DV_PLAN_COMPLETED
+### TESTPLAN_COMPLETED
 
-A fully completed DV plan written in Hjson, indicating the list of planned tests with descriptions indicating the goal of the test and optionally details on stimulus and the checking procedure.
-A fully completed functional coverage plan written in Hjson, indicating the list of functional coverage points and coverage crosses with a description of what feature is covered by this coverpoint.
+Testplan completely written (in HJson format) indicating:
+- Testpoints (a list of planned tests), each mapping to a design feature, with a description highlighting the goal of the test and optionally, the stimulus and the checking procedure.
+- The functional coverage plan captured as a list of covergroups, with a description highlighting what feature is expected to be covered by each covergroup. \
```

Comment:
Nit: I don't think the backslash is needed here. Markdown combines sentences within a bullet point based on indentation.

Reply:
Hmm, right. `\` forces a line-break (which is not needed here).